docs-ref-conceptual/spring-framework/deploy-spring-boot-java-app-with-maven-plugin.md

```diff
@@ -94,35 +95,47 @@ In this section, you will configure the Spring Boot project `pom.xml` so that Ma
 
    ```xml
    <plugin>
-    <groupId>com.microsoft.azure</groupId>
-    <artifactId>azure-webapp-maven-plugin</artifactId>
-    <version>1.4.0</version>
-    <configuration>
-      <deploymentType>jar</deploymentType>
-
-      <!-- configure app to run on port 80, required by App Service -->
-      <appSettings>
-        <property> 
-          <name>JAVA_OPTS</name> 
-          <value>-Dserver.port=80</value> 
-        </property> 
-      </appSettings>
-
-      <!-- Web App information -->
-      <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
-      <appName>${WEBAPP_NAME}</appName>
-      <region>${REGION}</region>  
-
-      <!-- Java Runtime Stack for Web App on Linux-->
-      <linuxRuntime>jre8</linuxRuntime>
-    </configuration>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-webapp-maven-plugin</artifactId>
+      <version>1.6.0</version>
+      <configuration>
+         <schemaVersion>V2</schemaVersion>
+         <subscriptionId>${SUBSCRIPTION_ID}</subscriptionId>
+         <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
+         <appName>${WEBAPP_NAME}</appName>
+         <region>${REGION}</region>
+         <pricingTier>P1V2</pricingTier>
+         <runtime>
+         <os>linux</os>
+         <javaVersion>jre8</javaVersion>
+         <webContainer>jre8</webContainer>
+         </runtime>
+         <appSettings>
+         <!-- Set port forwarding for app service-->
+         <property>
+            <name>PORT</name>
+            <value>8080</value>
+         </property>
+         </appSettings>
+         <deployment>
+         <resources>
+            <resource>
+               <directory>${project.basedir}/target</directory>
+               <includes>
+               <include>*.jar</include>
```

Comment:
Ideally, this should reference only ${project.build.finalName}.jar artifact.

Reply:
done